src/wp-content/themes/twentynineteen/style-editor.scss

```diff
@@ -846,9 +846,9 @@ ul.wp-block-archives,
 	}
 }
 
+/* Add style for galleries in classic-editor block */
```

Comment:
Maybe this comment could just be removed, it seems miss placed since the next code block is not for the galleries in the classic editor.

Reply:
Removing the comment entirely could be good since it's still under the Classic Editor section.

https://github.com/WordPress/twentynineteen/commit/0064cd59938eb50868486fbcac8bf15450e81f78 added the styles, copying the comment from the next ruleset (which was removed before version 1.0). If it is worth having a comment for the `blockquote` styles, the new explanation could come from the commit message: "Adjust blockquote styles within the classic editor block"

Reply:
I removed it.